framework/resource_router.go

```diff
@@ -37,10 +37,10 @@ func (r *ResourceRouter) ResourceSet(obj interface{}) (*ResourceSet, error) {
 	}
 
 	if len(found) == 0 {
-		return nil, microerror.Maskf(executionFailedError, "could not find any resource router")
+		return nil, microerror.Maskf(executionFailedError, "could not find handling resource set")
 	}
 	if len(found) > 1 {
-		return nil, microerror.Maskf(executionFailedError, "multiple resource routers not allowed")
+		return nil, microerror.Maskf(executionFailedError, "multiple handling resource sets found")
```

Comment:
Along the lines of above, my suggestion for this:
`multiple matches for resource set; only single allowed`

Reply:
Changing to `multiple handling resource sets found; only single allowed` to align with `handling resource set not found`.